models/bgrid_solo/work/workshop_setup.sh

```diff
@@ -39,12 +39,18 @@
 #----------------------------------------------------------------------
 
 # Build the DART programs 
-echo 'runing quickbuild.sh'
+echo 'running quickbuild.sh'
```

Comment:
comment for line 1. 
#!/bin/csh but the script extension is .sh

Reply:
I think that line 1 should be changed to \#!/bin/bash
and any additional commands use bash syntax.